src/app/pages/LandingPage/components/Promotions/components/PromotionsCarousel/CustomRightArrow.tsx

```diff
@@ -2,12 +2,16 @@ import React from 'react';
 import { ArrowProps } from 'react-multi-carousel';
 import { faChevronRight } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { StyledArrow } from './styled';
+import styles from './index.module.scss';
+import classNames from 'classnames';
 
 export const CustomRightArrow: React.FC<ArrowProps> = ({ onClick }) => {
   return (
-    <StyledArrow onClick={() => onClick?.()}>
+    <div
+      className={classNames(styles.arrow, 'tw-right-4')}
+      onClick={() => onClick?.()}
```

Comment:
would it work like this?
```suggestion
      onClick={onClick!}
```

Reply:
Well, it's even more simple :D A simple `onClick={onClick}` is sufficient, I guess I wanted to be overly cautious back when I implemented it :D

Reply:
@creed-victor please don't use or recommend the exclamation mark postfix it hard overrides the typecheck and can lead to serious issues.